src/Domain/Creators/Shell.hpp

```diff
@@ -49,6 +49,33 @@ class Shell : public DomainCreator<3> {
       CoordinateMaps::ProductOf2Maps<CoordinateMaps::Affine,
                                      CoordinateMaps::Identity<2>>>>;
 
+  /// Options for the EquatorialCompression map
+  struct EquatorialCompressionOptions {
```

Comment:
Here's another thought that I should have had earlier, but I only realized this now after looking at your code. Feel free to ignore since I'm coming up with this so late. I think we can just make the `EquatorialCompression` map option-creatable. Then you can use it directly in the `Options::Auto<>` below, store it in the `Shell` class as a `std::optional<>`, and pass it on to `sph_wedge_coordinate_maps` as such. There, you can do `equatorial_compression.value_or(EquatorialCompression{1., 2})` when you do the map-composition. Most code in tests etc can just pass `std::nullopt` instead having to deal with the parameters.

Again, please treat this as completely optional. Maybe I'm going a bit overboard here with suggesting better ways to code this, and we should just move on :)